app/src/main/java/sitsko/vlad/epamhomework/MainActivity.java

```diff
@@ -1,13 +1,120 @@
 package sitsko.vlad.epamhomework;
 
+import android.content.res.ColorStateList;
+import android.support.design.widget.NavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v4.widget.ImageViewCompat;
+import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.ImageView;
+
+import java.util.Random;
+
+import sitsko.vlad.epamhomework.compoundview.ProfileDataModel;
+import sitsko.vlad.epamhomework.compoundview.ProfileView;
+import sitsko.vlad.epamhomework.fragments.FirstFragment;
+import sitsko.vlad.epamhomework.fragments.SecondFragment;
 
 public class MainActivity extends AppCompatActivity {
 
+    private DrawerLayout drawerLayout;
+    private Toolbar toolbar;
+    private NavigationView navigationView;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        toolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        ActionBar actionbar = getSupportActionBar();
+        actionbar.setDisplayHomeAsUpEnabled(true);
+        actionbar.setHomeAsUpIndicator(R.drawable.ic_menu);
+
+        drawerLayout = findViewById(R.id.drawer_layout);
+
+        navigationView = findViewById(R.id.nav_view);
+        navigationView.setNavigationItemSelectedListener(
+                new NavigationView.OnNavigationItemSelectedListener() {
```

Comment:
too big unnamed inner class.
Clean code rules always should be applied, not only for HW Clean code